ext/oboe_metal/extconf.rb

```diff
@@ -118,22 +124,27 @@
     $libs = append_library($libs, 'stdc++')
 
     $CFLAGS << " #{ENV.fetch('CFLAGS', nil)}"
-    # $CPPFLAGS << " #{ENV['CPPFLAGS']} -std=c++11"
-    # TODO for debugging: -pg -gdwarf-2, remove for production
-    # -pg does not work on alpine https://www.openwall.com/lists/musl/2014/11/05/2
-    $CPPFLAGS << " #{ENV.fetch('CPPFLAGS', nil)} -std=c++11  -gdwarf-2 -I$$ORIGIN/../ext/oboe_metal/include -I$$ORIGIN/../ext/oboe_metal/src"
-    # $CPPFLAGS << " #{ENV['CPPFLAGS']} -std=c++11 -I$$ORIGIN/../ext/oboe_metal/include"
+
+    # -pg option is used for generating profiling information with gprof
```

Comment:
Good to know, although it's intentionally? not set anywhere below... while `gdwarf-2` is set if OBOE_DEBUG because that's actually required for extended debug info, yes?

Reply:
Yes, `-pg` option is only for profiling which we don't support; but I was thinking to keep here so I know the starting point of having profiling in the future.

`gdwarf-2` is kind required for debug, but I believe it's mainly for formatting.